src/internal/m365/collection/drive/collections_test.go

```diff
@@ -220,6 +220,30 @@ func (suite *OneDriveCollectionsUnitSuite) TestPopulateDriveCollections() {
 			expectedExcludes:         map[string]struct{}{},
 			expectedTopLevelPackages: map[string]struct{}{},
 		},
+		{
+			name: "Single Folder created twice", // deleted a created with same name in between a backup
+			items: []models.DriveItemable{
+				driveRootItem("root"),
+				driveItem("id1", "folder", testBaseDrivePath, "root", false, true, false),
```

Comment:
do we have any indication that this is what's actually being returned from graph? It's nice to be able to handle it if we absolutely need to, but I'm also a bit worried that it could somehow hide other things if we're not careful

Reply:
+1 here.  Seems reasonable that this is the response we get, but worth validating.

Reply:
discussed out of band, does indeed seem like what we're getting back